packages/p2p/src/components/modal-manager/modals/share-my-ads-modal/share-my-ads-card/share-my-ads-card.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import { QRCode } from 'react-qrcode-logo';
+import { QRCodeSVG } from 'qrcode.react';
```

Comment:
changed libraries as the previous one cause downloading and scanning issues on responsive:
![sell_57](https://github.com/ameerul-deriv/deriv-app/assets/103412909/df6813dd-4cb4-4ec6-81eb-7cd9e7039305)